go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/talos-systems/talos
 go 1.16
 
 replace (
+	github.com/talos-systems/go-blockdevice => github.com/Unix4ever/go-blockdevice v0.1.1-0.20210526144506-0573ad3d7b15
```

Comment:
should this be removed in favor of merged PR?

Reply:
oh, right. I though I did that already